src/store/page/page-element.ts

```diff
@@ -150,7 +170,13 @@ export class PageElement {
 		if (json && (json as JsonObject)['_type'] === 'pattern') {
 			return PageElement.fromJsonObject(json as JsonObject, store, this);
 		} else {
-			const element: PageElement = new PageElement(store.getPattern('text'), false, this);
+			const styleguide = store.getStyleguide('synthetic') as Styleguide;
```

Comment:
`getStyleguide` returns something with the type `Styleguide`. You should not need to typecast the result